app/Track.php

```diff
@@ -214,7 +232,8 @@ public static function mapPublicTrackShow($track)
                 'name' => $trackFile->format,
                 'extension' => $trackFile->extension,
                 'url' => $trackFile->url,
-                'size' => $trackFile->size
+                'size' => $trackFile->size,
+                'isCacheable' => $trackFile->is_cacheable
```

Comment:
The `size` and `isCacheable` attributes here should be casted explicitly to the correct types.